src/main/java/edu/harvard/iq/dataverse/api/AbstractApiBean.java

```diff
@@ -661,7 +635,15 @@ protected Response ok( JsonArrayBuilder bld ) {
             .add("data", bld).build())
             .type(MediaType.APPLICATION_JSON).build();
     }
-    
+
+    protected Response ok( JsonArrayBuilder bld , long totalCount) {
+        return Response.ok(Json.createObjectBuilder()
+                        .add("status", ApiConstants.STATUS_OK)
+                        .add("total_count", totalCount)
```

Comment:
Although the search endpoint uses underscore for "total_count", I would use camel case here ("totalCount") as it is the convention we normally use in most endpoints.

Reply:
Fixed